src/util/virutil.c

```diff
@@ -277,6 +277,33 @@ virParseVersionString(const char *str, unsigned long *version,
     return 0;
 }
 
+int
+virCHVersionString(const char *str, unsigned long *version,
+                      bool allowMissing)
+{
+    unsigned int major, minor = 0, micro = 0;
+
+    if (strncmp(str, "cloud-hypervisor v", 18) == 0) {
+        // check for regular 'cloud-hypervisor --version' output
```

Comment:
Guess should use `C` style comment here.

Thanks

Reply:
@weltling Done